blocks/rich-text/index.js

```diff
@@ -92,6 +93,7 @@ export default class RichText extends Component {
 		this.getSettings = this.getSettings.bind( this );
 		this.onSetup = this.onSetup.bind( this );
 		this.onChange = this.onChange.bind( this );
+		this.throttledOnChange = throttle( this.onChange.bind( this ), 500 );
```

Comment:
If we can, I'd love if we would avoid throttling. Case in point: We're not cancelling a scheduled change when the component unmounts.

Reply:
Yep, if the performance fix is enough, I'm fine dropping it.

Reply:
I tried dropping it but I think performance-wise it's slightly better if we keep it

Reply:
It seems like it always dispatched change when I type the first character. Should we wait until 500ms passes instead?

Reply:
@youknowriad What can be do to get rid of this? :)